RadarSDK.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name                  = 'RadarSDK'
-  s.version               = '3.3.0'
+  s.version               = '3.3.1'
   s.summary               = 'iOS SDK for Radar, the leading geofencing and location tracking platform'
   s.homepage              = 'https://radar.com'
   s.author                = { 'Radar Labs, Inc.' => 'support@radar.com' }
   s.platform              = :ios
   s.source                = { :git => 'https://github.com/radarlabs/radar-sdk-ios.git', :tag => s.version.to_s }
-  s.vendored_frameworks   = 'dist/RadarSDK.xcframework'
+  s.source_files          = ["RadarSDK/*.{h,m}", "RadarSDK/Internal/*.{h,m}", "RadarSDK/Include/*.h"]
```

Comment:
We can remove `"RadarSDK/Internal/*.{h,m}"` in a follow up release (harmless)

Reply:
Ooh, good catch.